### PR TITLE
Update Hibernate ORM 5.4.24.Final #1023

### PR DESCRIPTION
**Pull request by btkobayashirun**
Please review #1023.

### CODE REVIEW

File: terasoluna-gfw-parent/pom.xml [hunk 1]
```diff
@@ -344,6 +344,14 @@
       </dependency>
       <!-- == End MyBatis == -->
 
+      <!-- == Begin Hibernate == -->
+      <dependency>
+        <groupId>org.hibernate</groupId>
+        <artifactId>hibernate-core</artifactId>
+        <version>${org.hibernate.version}</version>
+      </dependency>
```
Comment (yoshikawaa) [inline terasoluna-gfw-parent/pom.xml:352]: Why don't you have the same version of `hibernate-entitymanager`?
I think... considering the users who are still referencing via HEM, misalignment will cause confusion.

**Review by yoshikawaa — COMMENTED** [review]
@takahashihrzg @btkatoufm Please check.

File: terasoluna-gfw-parent/pom.xml (hunk 1 above)
Comment (btkobayashirun) [reply terasoluna-gfw-parent/pom.xml:352]: add dependencyManagement definition for entity-manager.
8508d2eed071c61f267993f2dc8b33d4f6f426ac in #1025

File: terasoluna-gfw-parent/pom.xml [hunk 2]
```diff
@@ -534,6 +542,8 @@
     <!-- == MyBatis3 == -->
     <org.mybatis.version>3.5.5</org.mybatis.version>
     <org.mybatis.spring.version>2.0.5</org.mybatis.spring.version>
+    <!-- == Hibernate == -->
+    <org.hibernate.version>5.4.24.Final</org.hibernate.version>
```
[the remark this answers is not in the document]
Comment (btkobayashirun) [reply terasoluna-gfw-parent/pom.xml:546]: added a commnet.
363844dc9521c9074039183f1d3f9dc9eeb53995 in #1025

**Pull request merged**